### PR TITLE
Feature levelfeatures

### PR DESCRIPTION
**Pull request by LeonWubben**
Decreases level dimensions at level 10, 20 and 30.
Increases amount of colors at level 5, 15 and 25

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/semgr8/model/piece/Gem.java [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
+
     /**
-     * The number of Colors.
-     */
-    public static final int AMOUNT_OF_COLORS = 6;
-    /**
-     * The color of this Gem, can be between 0 and 5 (inclusive).
+     * The color of this Gem, can be between 0 and MAX_AMOUNT_OF_COLORS-1 (inclusive).
      */
     protected int color;
```
Comment (albert-smit) [inline src/main/java/nl/tudelft/ti2206/semgr8/model/piece/Gem.java]: this should really be private with a protected setter

**Comment by albert-smit** [comment]
looks good, but please fix the last warnings from the static analysis tools and run the auto formatter

File: src/test/java/nl/tudelft/ti2206/semgr8/model/LevelTest.java [hunk 2]
```diff
@@ -82,6 +83,6 @@ public void testNextLevelProgressLevel2() {
      */
     @Test
     public void testToString() {
-        assertEquals( "Level{level=1}",level.toString());
+        assertEquals("Level{level=1}", level.toString());
     }
 }
```
Comment (albert-smit) [inline src/test/java/nl/tudelft/ti2206/semgr8/model/LevelTest.java]: missing newline

**Comment by kiliangrashoff** [comment]
Please note that this branch failed to build due to travis failing to pull it. As you can see on the merge commit, it did actually build properly.

**Pull request merged**